src/main/java/com/coveros/selenified/services/HTTP.java

```diff
@@ -378,8 +378,8 @@ private Response getResponse(HttpURLConnection connection) {
         JsonArray array;
         String data;
         BufferedReader rd = null;
-        try { // NOSONAR - unable to use the try-with-resources block, as the rd
-            // needs to be read in the finally, and can't be closed
+        // unable to use the try-with-resources block, as the rd needs to be read in the finally, and can't be closed
+        try {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Change this "try" to a try-with-resources. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS2093)